cmd/geoipupdate/main.go

```diff
@@ -47,7 +41,8 @@ func main() {
 		geoipupdate.WithOutput(args.Output),
 	)
 	if err != nil {
-		fatalLogger("error loading configuration", err)
+		log.Printf("Error loading configuration: %s", err)
```

Comment:
`log.Fatalf` calls `os.Exit(1)` https://pkg.go.dev/log#Logger.Fatalf

Reply:
Thanks. Yeah, I just took the code from the old logger without thinking about it much.